output/text.go

```diff
@@ -95,7 +95,17 @@ func Text(results []*comparer.Result, includePassing bool, tweaks []*config.Quer
 	fmt.Println(strings.Repeat("=", 80))
 	fmt.Printf("Total: %d / %d (%.2f%%) passed, %d unsupported\n", successes, len(results), 100*float64(successes)/float64(len(results)), unsupported)
 
-	fmt.Printf("Duration prom: %v, ref: %v\n", impl1Duration, impl2Duration)
+	fmt.Printf("Duration Ref prom: %v, Kfuse: %v\n", impl1Duration, impl2Duration)
+
+	if targetTotalDuration != "" {
+		duration, err := time.ParseDuration(targetTotalDuration)
+		if err != nil {
+			fmt.Printf("Error: Unable to parse input target total duration")
+		}
+		if impl2Duration > duration {
+			fmt.Printf("Kfuse duration %v exceeded target total duration %v\n", impl2Duration, duration)
```

Comment:
Should we have this per query or overall? Here it seems we have it overall?

Reply:
I think we discussed yesterday and we'll do overall for now, we can update later to add per query too.

Reply:
It'd be almost impossible to debug or have anything meaningful out of this.. we do have individual query durations as part of the result.. should we log those as well when this happens? Ideally, we'd like to know which query took how long etc.. and somehow I feel that it's better to have a per query limit rather than over all limit.

Reply:
I think it's hard to set per query limit (since there are variety of ranges/steps, etc), or just have a single global per query limit right now (instead of a global total test query time limit) (e.g., 1s), and just flag it if a query took more than that?

Reply:
I understand that due to the variability of so many things.. it's difficult to drive per query upper limit from config.. but I don't see how the overall query duration will help with any debugging.. if we agree on that then we can see what else can be done.. if not, do explain what the thought is on the usefulness of just overall duration.. 

one thought for per query: 
since the template is driving different query parameters/types.. can we atleast have a limit for per template type? and this could be the upper limit bsaed on what possible queries that template generates..  it's not going to be easy to tune this, I do understand.

Reply:
Let's push this in as is for now and we can worry about enhancing it more later. This is supposed to be a simple guard. Its not a perf test and we will need more thought before we can have something like that.